kolibri/plugins/learn/assets/src/views/content-page/index.vue

```diff
@@ -182,17 +182,11 @@
         }
       },
       showNextBtn() {
-        return this.progress >= 1 && this.content && this.nextContentLink;
+        return this.sessionProgress >= 1 && this.content && this.nextContentLink;
```

Comment:
This is incorrect. Should be summaryProgress, not sessionProgress.

Also, this probably should have been conditionalized for anonymous users as well?